lib/engine/depot.rb

```diff
@@ -111,6 +111,8 @@ def discountable_trains_for(corporation)
           discount_info
         end.compact
       end
+      other_discounts = @game.discountable_trains_for(corporation)
```

Comment:
move this whole method there. this is not providing much benefit if people want to override the top part,  also now there is a performance hit (probably negligble) of creating new arrays with discounts + []